### DISCUSSION TITLE
Gathering Some new ideas for the Website Enhancement

### DISCUSSION THREAD

**Discussion by ayan-biswas0412** [ayan-biswas0412/gssoc2021-HotelOnTouch#92] [Ideas]
please go through this issue and let's make something cool https://github.com/ayan-biswas0412/gssoc2021-HotelOnTouch/issues/90

**Comment by Vi1234sh12** [comment]
I'm Vishal Dhanure  I created a design on Figma  this link you can see https://www.figma.com/file/6EgUU72MvvJFHqb9eNYjYx/HotelOnTouchMain?node-id=0%3A1 

if any problem please let me know or adding features

thanks 
Vishal Dhanure

**Reply by ayan-biswas0412** [reply]
hi thank you for investing your time but I would like to give you some comments 
1. we are not making a hotel booking platform
2. but our platform's work starts from when a guest checkins to the hotel
3. we want to make a hotel automation system like online booking of laundry and restaurant services etc please see the wiki section of this project for more info 

Please make your designs according to that and we will value your contributions with topmost priority

**Comment by ayan-biswas0412** [comment]
I am looking for a new maintainer for this project, interested candidates may mail their cv at ayan.biswas@judh.in
with their cv (please only provide your development background)

Mail Subject: Application for Maintainers | HotelOnTouch
